src/sidebar.component.ts

```diff
@@ -76,8 +75,7 @@ import { isBrowser, upperCaseFirst, isLTR, isIOS } from './utils';
       transition: transform 0.3s cubic-bezier(0, 0, 0.3, 1);
     }
   `],
-  changeDetection: ChangeDetectionStrategy.OnPush,
-  encapsulation: ViewEncapsulation.None
```

Comment:
The users needs to use the deep (>>>) on css to modify the style of the sidebar if the css file is loaded by angular. On other cases, the deep selector is not necessary.